src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppBase.cs

```diff
@@ -75,22 +75,13 @@ public async Task OnStartProcess(string startEvent, Instance instance)
         {
             await Task.CompletedTask;
             _logger.LogInformation($"OnStartProcess for {instance.Id}");
-
         }
 
         /// <inheritdoc />
         public async Task OnEndProcess(string endEvent, Instance instance)
         {
+            await Task.CompletedTask;
             _logger.LogInformation($"OnEndProcess for {instance.Id}, endEvent: {endEvent}");
-
-            if (endEvent != null && endEvent.Equals("EndEvent_1"))
-            {
-                await _pdfService.GenerateAndStoreReceiptPDF(instance);
-            }
-
-            // Set archived status
-            instance.Status ??= new InstanceStatus();
```

Comment:
Is it deliberate to remove this instance update?

Reply:
Jupp. The values are set in storage. Not sure this was ever exposed to the end user. as the request to trigger this is Process/next and that only returns the process state and not the instance state